bool is_enemy_recognized
float32 enemy_dist
int64 enemy_direct 
bool is_enemy_marker_recognized
int64 enemy_marker_direct